silq/parameters/acquisition_parameters.py

```diff
@@ -1858,17 +1860,19 @@ def ESR_frequencies(self, ESR_frequencies: List):
         self.ESR['ESR_pulses'] = updated_ESR_pulses
 
     def analyse(self, traces: Dict[str, Dict[str, np.ndarray]] = None):
-        """Analyse flipping events between nuclear states
+        """Analyse flipping events between nuclear states and determine nuclear state
 
         Returns:
             (Dict[str, Any]): Dict containing:
 
-            :results_read (dict): `analyse_traces` results for each read
+            * **results_read** (dict): `analyse_traces` results for each read
```

Comment:
Is there a reason you're reformatting the docstrings like this?

Reply:
When I was using the same formatting style that it already had, Serwan said I should use different one, which I found elsewhere, so now I have no idea which style to use...